about.css

```diff
@@ -0,0 +1,444 @@
+
+.course-h {
+  font-size: 30px;
+  font-weight: 700;
+}
+
+.course-description {
+  line-height: 26px;
+}
+
+.course-item img {
+  display: block;
+  width: 100%;
+}
+
+.partner {
+  background-color: var(--primary-color-1);
+  color: var(--primary-color-3);
+  padding: 10px;
+  text-align: center;
+}
+
+.partner hr {
+  color: var(--primary-color-2);
+  border: none;
+  border-top: 1px solid var(--primary-color-2);
+  margin: 10px auto;
+  width: 20px;
+}
+
+.partner-logo ul {
+  display: flex;
+  flex-wrap: wrap;
+  list-style: none;
+  justify-content: center;
+  padding: 0;
+}
+
+.partner-logo ul li img {
+  margin: 10px;
+  height: 14px;
+  width: auto;
+}
+
+footer {
+  display: flex;
+  gap: 30px;
+  align-items: center;
+  padding: 20px 40px;
+}
+
+@media only screen and (min-width: 768px) {
+  .hamberger {
+    visibility: hidden;
+  }
+
+  .user-acc {
+    display: flex;
+    justify-content: flex-end;
+    background-color: var(--primary-color-1);
+    align-items: center;
+    padding: 0.3rem 5rem;
+  }
+
+  .nav-user > ul {
+    display: flex;
+  }
+
+  .menu > li {
+    padding: 0 20px;
+  }
+
+  .nav-user > ul > li {
+    padding: 5px 10px;
+    color: #fff;
+  }
+
+  .social-icon > i {
+    padding: 5px 10px;
+    color: #fff;
+  }
+
+  .main-menu {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 1rem 9rem;
+    box-shadow: rgba(0, 0, 0, 0.1) 0 10px 15px -3px, rgba(0, 0, 0, 0.05) 0 4px 6px -2px;
+  }
+
+  .logo {
+    display: block;
+    width: 64px;
+    margin-right: -20px;
+  }
+
+  .menu {
+    display: flex;
+    justify-content: space-around;
+    font-weight: 400;
+  }
+
+  .main-nav {
+    display: flex;
+    align-items: center;
+  }
+
+  .menu > li > a {
+    text-decoration: none;
+    color: var(--primary-color-1);
+  }
+
+  .menu > li > a:active {
+    color: var(--primary-color-2);
+  }
+
+  .event {
+    display: block;
+    border: 2px solid var(--primary-color-2);
+    color: var(--primary-color-2);
+    cursor: pointer;
+    font-weight: 700;
+    padding: 10px 20px;
+    margin-left: 40px;
+    text-decoration: none;
+  }
+
+  .header {
+    padding: 5rem 9rem;
+    text-align: center;
+  }
+
+  .header > h1 {
+    font-size: 48px;
+    text-align: center;
+    padding: 0;
+    margin: 0;
+  }
+
+  .header > hr {
+    border: none;
+    border-top: 2px solid var(--primary-color-2);
+    width: 20px;
+    margin: 20px auto;
+  }
+
+  .sub-header {
+    display: none;
+  }
+
+  .description-h {
+    max-width: 680px;
+    margin: 15px auto;
+  }
+
+  .emphasic > img {
+    max-width: 240px;
+    margin: 0 auto;
+    margin-top: 30px;
+  }
+
+  .course-items {
+    display: flex;
+    justify-content: center;
+  }
+
+  .course-item {
+    max-width: 400px;
+  }
+
+  .partner {
+    display: none;
+  }
+
+  footer {
+    background-color: var(--primary-color-1);
+    color: #fff;
+    margin-top: 100px;
+    padding: 3rem 11rem 3rem 12rem;
+    gap: 30px;
+  }
+}
```

Comment:
- [ ] Great work adjusting the spaces between elements in the **_Footer_**. However, when you adjusted them, the elements became more aligned to the left instead of being aligned to the center as seen in the template design. I would advise that you align them to the center. There is no need to touch the spacing between elements, but there is a need to align them as a component to the center. 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/211145270-7ef42c46-79a9-48c6-9e8b-f1883dd6a98e.png)

### Template Design

![image](https://user-images.githubusercontent.com/77106962/211145313-39ce7cf1-2e82-4460-a4b0-ddc791211d50.png)